### PR TITLE
Add scripts to generate intl_translation in CI

### PR DESCRIPTION
**Pull request by hoangdat**
@tnstung @manhduc-nguyen @imGok Please review this PR.

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -61,6 +61,11 @@ workflows:
             cd data
             flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs
 
+            # Generate intl translation
+            cd ..
+            flutter pub pub run intl_translation:extract_to_arb --output-dir=lib/l10n lib/presentation/localizations/app_localizations.dart
+            flutter pub pub run intl_translation:generate_from_arb --output-dir=lib/l10n \ --no-use-deferred-loading lib/presentation/localizations/applocalizations.dart lib/l10n/intl*.arb
```
Comment (imGok) [inline bitrise.yml:67]: Why is there 2 pub ? `flutter pub pub`

Comment (hoangdat) [reply bitrise.yml:67]: my false :(

Comment (imGok) [inline bitrise.yml:67]: I think `applocalizations.dart ` should be `app_localizations.dart`

File: bitrise.yml [hunk 2]
```diff
@@ -61,6 +61,11 @@ workflows:
             cd data
             flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs
 
+            # Generate intl translation
```
[the remark this answers is not in the document]
Comment (tnstung) [reply bitrise.yml]: Let me optimize this.

**Review by tnstung — APPROVED** [review]

File: bitrise.yml [hunk 3]
```diff
@@ -58,11 +58,19 @@ workflows:
...
-            cd data
-            flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs
+            flutter pub get
+            flutter pub get domain
+            flutter pub get data
+
+            cd $BITRISE_FLUTTER_PROJECT_LOCATION/data
+            flutter pub run build_runner build --delete-conflicting-outputs
+
+            cd $BITRISE_FLUTTER_PROJECT_LOCATION
```
Comment (hoangdat) [inline bitrise.yml:68]: @tnstung are you sure we `cd` to project root folder with this `cmd`

**Pull request merged**